lib/parsers/entity_parser.js

```diff
@@ -274,6 +275,15 @@ function setValidationsOfField(jdlField, fieldData) {
   }
 }
 
+function setOptionsOfField(jdlField, fieldData) {
+  if (Object.entries(jdlField.options).length !== 0) {
+    fieldData.options = {};
+    Object.entries(jdlField.options).forEach(option => {
```

Comment:
What you can do here:
```javascript
Object.entries(jdlField.options).forEach(([key, value]) => ...)
```

Reply:
indeed destructing is much better, doing this